### DISCUSSION TITLE
find clipping tool

### DISCUSSION THREAD

**Discussion by ghost** [audacity/audacity#1987] [General]
How to use the cliping tool in AUDACITY?

**Comment by SteveDaulton** [comment]
Do you mean [Clip Fix](https://manual.audacityteam.org/man/clip_fix.html)?

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/92734176/138181737-61a32ca3-64a4-47ec-851e-496251bab025.png)

**Comment by SteveDaulton** [comment]
See here: https://manual-audacityteam-org.translate.goog/man/find_clipping.html?_x_tr_sl=en&_x_tr_tl=pt&_x_tr_hl=en-GB&_x_tr_pto=nui

**Reply by ghost** [reply]
I did not understand. A bunch of numbers appear in a new strip below. But, clippings are found and then what? What I do? Normalize height, reduce volume, amplify creates clipping, what do I do?
![image](https://user-images.githubusercontent.com/92734176/138270202-eeb01462-ff5d-43b7-81c3-936a139bee44.png)

**Comment by SteveDaulton** [comment]
The "4 of 4" means that a sequence of 4 samples have been found that are greater or equal to 0 dB.

In this example (below) I have zoomed in vertically and horizontally (See: https://manual.audacityteam.org/man/zooming.html) to inspect the indicated clipping. A sequence of 9 samples have been detected, 8 of which are greater or equal to 0 dB.
I can then decide what to do about it - for example, I could ignore it, or I could try to repair it with the "Repair" effect (See: https://manual.audacityteam.org/man/repair.html), or I could try to repair it with the "Draw" tool (See: https://manual.audacityteam.org/man/draw_tool.html)

Please note that this (GitHub Discussions) is not a help channel. It is for discussing software development issues.
There are two help channels for Audacity:

- The Audacity Help Forum: https://forum.audacityteam.org
- A help channel on Discord: https://discord.gg/JBANy7ju

![Tracks000](https://user-images.githubusercontent.com/11681097/138271988-f16d38bd-e318-4b73-8e30-d272021071f2.png)
